ui/packages/ui/src/app/pages/createConnector/connectorSteps/FilterConfigComponent.tsx

```diff
@@ -39,36 +30,9 @@ export interface IFilterConfigComponentProps {
   filterValues: Map<string, string>;
   connectorType: string;
   updateFilterValues: (data: Map<string, string>) => void;
-  setIsValidFilter: (val:SetStateAction<boolean>) => void;
-  parent: string;
-  child: string;
-  parentExcludeList: string;
-  parentIncludeList: string;
-  childExcludeList: string;
-  childIncludeList: string;
-  i18nFilterParentInfoMsg: string;
-  i18nFilterChildInfoMsg: string;
-  i18nFilterConfiguration: string;
-  i18nFilterPageHeadingText: string;
-  i18nFilterParentLabel: string;
-  i18nFilterChildLabel: string;
-  i18nFilterParentHelperText: string;
-  i18nFilterChildHelperText: string;
-  i18nInclude: string;
-  i18nExclude: string;
-  i18nApply: string;
-  i18nClearFilters: string;
-  i18nInvalidFilterText: string;
-  i18nMatchingFilterExpMsg: string;
-  i18nNoMatchingFilterExpMsg: string;
-  i18nClearFilterText: string;
-  i18nFilterExpressionResultText: string;
-  i18nCancel: string;
-  i18nClear: string;
-  i18nClearFilterConfMsg: string;
-  i18nNoMatchingTables: string;
-  i18nInvalidFilters: string;
-  i18nInvalidFilterExpText: string;
+  setIsValidFilter: (val: SetStateAction<boolean>) => void;
+  selectedConnectorType: string;
+  filtersArray: string[];
```

Comment:
filtersArray isn't used - you can remove it.  And then no need to pass in from CreateConnectorComponent.

Reply:
done